trade-finance-manager-ui/server/controllers/case/cancellation/validation/validate-bank-request-date.test.ts

```diff
@@ -84,12 +84,9 @@ describe('validate bank request date', () => {
         // Assert
         const expectedMessage = 'The bank request date cannot exceed 12 months in the future from the submission date';
 
-        const expectedError = {
-          summary: [{ text: expectedMessage, href: '#bank-request-date-day' }],
-          bankRequestDateError: { message: expectedMessage, fields: ['bank-request-date-day', 'bank-request-date-month', 'bank-request-date-year'] },
-        };
+        const expected = getErrorObjectFromMessageAndRefs(expectedMessage, ['bank-request-date-day', 'bank-request-date-month', 'bank-request-date-year']);
 
-        expect(result.errors).toEqual(expectedError);
+        expect(result).toEqual(expected);
```

Comment:
@ttbarnes apologies somehow this didn't get pushed on the last PR - have made the change here and in the new test files :)